d2/src/main/java/com/linkedin/d2/xds/XdsToD2PropertiesAdaptor.java

```diff
@@ -357,26 +358,20 @@ public void onReconnect()
     };
   }
 
-  private void updateSymlinkAndActualNodeMap(String symlinkName, String actualNodeName)
-  {
-    synchronized (_symlinkAndActualNodeLock)
-    {
+  private void updateSymlinkAndActualNodeMap(String symlinkName, String actualNodeName) {
```

Comment:
the "{" should be on the next line. Applies to all similar places in the PR.

Reply:
Thanks. Format the code